### PR TITLE
deprecate bits to bitstring

### PR DESCRIPTION
**Pull request by Sacha0**
This pull request deprecates `bits` to `bitstring`, addressing #24263. Best!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -556,8 +556,9 @@ Deprecated or removed
     `ReinterpretArray`. The three argument form of `reinterpret` that implicitly reshapes
     has been deprecated ([#23750]).
 
-  * `num2hex` and `hex2num` have been deprecated in favor of `reinterpret` combined with `parse`/`hex`
-    ([#22088])
+  * `bits` has been deprecated in favor of `bitstring` ([#WHATAREBIRDS]).
```
Comment (fredrikekre) [inline NEWS.md]: `([#24281])`

File: base/deprecated.jl [hunk 2]
```diff
@@ -1913,6 +1913,9 @@ end
 @deprecate float(x::AbstractString) parse(Float64, x)
 @deprecate float(a::AbstractArray{<:AbstractString}) parse.(Float64, a)
 
+# deprecate bits to bitstring
+@deprecate bits(x) bitstring(x)
```
Comment (fredrikekre) [inline base/deprecated.jl]: I suppose this could be `@deprecate bits bitstring`?

**Review by fredrikekre — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (JeffBezanson) [reply NEWS.md]: or `#24263`

Comment (Sacha0) [reply NEWS.md]: Added both. Thanks!

File: base/deprecated.jl (hunk 2 above)
Comment (Sacha0) [reply base/deprecated.jl]: Good call! Simplified. Thanks! :)

**Comment by Sacha0** [comment]
Thanks all! :)

**Pull request merged**